aggregator/src/main/java/com/redhat/cloud/notifications/db/EmailAggregationResources.java

```diff
@@ -0,0 +1,62 @@
+package com.redhat.cloud.notifications.db;
+
+import com.redhat.cloud.notifications.models.EmailAggregation;
+import com.redhat.cloud.notifications.models.EmailAggregationKey;
+import org.hibernate.Session;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.logging.Logger;
+
+@ApplicationScoped
+public class EmailAggregationResources {
+
+    private final Logger log = Logger.getLogger(this.getClass().getName());
+
+    @Inject
+    Session session;
+
+    @Transactional
+    public Boolean addEmailAggregation(EmailAggregation aggregation) {
+        try {
+            session.persist(aggregation);
```

Comment:
Just to clarify, this is the aggregated data?

Reply:
Yes. 

Not sure if important, but this is exactly the same as in notfications-backend.